### PR TITLE
zh-cn(translate): compression and decompression

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

**Review by JasonLamv-t — APPROVED** [review]
LGTM.

File: files/zh-cn/web/api/compressionstream/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: CompressionStream
+slug: Web/API/CompressionStream
+---
+
+{{APIRef("Compression Streams API")}}
+
+{{domxref('Compression Stream API','','',' ')}} 的 **`CompressionStream`** 接口是一个用于压缩数据流的 API。
```
Comment (yin1999) [inline files/zh-cn/web/api/compressionstream/index.md:8]:
```suggestion
{{domxref('Compression Streams API', '', '', 1)}} 的 **`CompressionStream`** 接口是一个用于压缩数据流的 API。
```

File: files/zh-cn/web/api/decompressionstream/index.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: DecompressionStream
+slug: Web/API/DecompressionStream
+---
+
+{{APIRef("Compression Streams API")}}
+
+{{domxref('Compression Stream API','','',' ')}} 接口的 **`DecompressionStream`** 是一个用与解压缩数据流的 API。
```
Comment (yin1999) [inline files/zh-cn/web/api/decompressionstream/index.md:8]:
```suggestion
{{domxref('Compression Streams API','','',1)}} 的 **`DecompressionStream`** 接口是一个用与解压缩数据流的 API。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/compressionstream/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/api/compressionstream/index.md:8]: image

File: files/zh-cn/web/api/decompressionstream/index.md (hunk 2 above)
Comment (fwqaaq) [reply files/zh-cn/web/api/decompressionstream/index.md]: oh, 搞错了，第二个参数才是描述

**Pull request merged**